.github/workflows/ci.yml

```diff
@@ -48,7 +55,13 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       matrix:
-        nxf_ver: ["21.10.4", ""]
+        include:
+          # Test pipeline minimum Nextflow version
+          - NXF_VER: '21.10.3'
```

Comment:
Why nextflow version 21.10.3 instead of 21.10.4?

Reply:
Ah right I mean this is the minimal version that should be tested, then the latest is also tested. So the minimal NXF_VER that still works for the pipeline should be cited here